server/src/services/user.service.ts

```diff
@@ -25,5 +39,38 @@ export class UserService extends CrudService<UserRepository> {
     if (!user) throw new BadRequestError()
     return user;
   }
+
+  async uploadProfilePicture(req: Request, res: Response) {
+    const userId = req.user!.id;
+    const { mimetype } = req.query;
+
+    if (!mimetype || typeof mimetype !== 'string') {
+      throw new BadRequestError('Mimetype is required');
```

Comment:
use 
```suggestion
      throw new BadRequestError(req.t('validation.isRequired',{field: "mime_type"}));
```
also update the [translation.json](https://github.com/aritradevelops/hush/blob/main/server/locales/en/translation.json) add under validation => `"isRequired": "{{field}} is required"`